test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -35,7 +35,43 @@ def setup
       verification_value: '737',
       month: 10,
       year: 2028)
-    @apple_pay = apple_pay_payment_token
+
+    @google_pay = network_tokenization_credit_card(
+      '4242424242424242',
+      payment_cryptogram: 'reddelicious',
```

Comment:
🛑  Warning:

Even when this works, I think is a better idea to use a random Base64 string, is not clear at all that just out of luck 'reddelicious' is a valid Base64 encoded string.